extension-androidauto/src/main/java/com/mapbox/maps/extension/androidauto/MapboxCarMapGestureHandler.kt

```diff
@@ -68,4 +68,15 @@ interface MapboxCarMapGestureHandler {
   ) {
     // Optional override
   }
+
+  /**
+   * TODO docs
```

Comment:
```suggestion
   * Allows you to observer map click gestures. The surface is
   * [MapboxCarMapObserver.onAttached] before this can be triggered.
   * 
   * @see [SurfaceCallback.onClick] for instructions to enable.
   * This method may not be called in some car systems.
   * 
   * @param x coordinate of the click in pixels
   * @param y coordinate of the click in pixels
```